### ISSUE TITLE
Running simulation on HPCC

### ISSUE THREAD

**Issue by tony1747** [MathCancer/PhysiCell#150]
I've run a few simulations on my laptop and everything works fine. Now I am trying to use the HPCC at my university to test the simulation, but I encountered this error while viewing the output images. 
![snapshot00000352](https://user-images.githubusercontent.com/64659546/231294917-7d480c7e-aa70-4d2e-81aa-cded67220e94.svg)

I have no idea what this error means exactly. I also notice that for the first few minutes of the simulation, there is nothing wrong with the images, however, this error message comes up in the middle of the simulation after a while. In the meantime, this error doesn't stop my simulation, instead, it keeps running till the end but the rest of the images are all having the same issue.

**Comment by rheiland** [comment]
We would definitely need more information about your situation to even begin to try to help you debug it. From your error, I'm guessing your .svg files have been corrupted - either they weren't completely written/saved or there's a memory error or something else. You'd want to be sure you haven't run up against limits on your HPC account, like storage limits for your output files or runtime limits, although you claim that your simulation continues to run. A few suggestions/questions:

* if you're willing to share your model, we could try to run it to see if by chance we could reproduce the problem
* you're running with identical parameters, i.e., the very same .xml configuration file, on both your laptop and the HPC?
* just to confirm, you're not trying to run it as a distributed memory program, e.g., using MPI, on the HPC are you?
* do you have any insights into how the HPC does file I/O? Would it be possible that output files are being written onto some unusual disk system where such I/O problems have appeared for others (probably unlikely)
* have you attempted to describe your problem to the HPC support staff? They may have some insights and suggestions.

**Comment by tony1747** [comment]
[cancer_immune.zip](https://github.com/MathCancer/PhysiCell/files/11216935/cancer_immune.zip)
[coor_anti.txt](https://github.com/MathCancer/PhysiCell/files/11216938/coor_anti.txt)
I have included both my codes and the data I used here. To answer your questions

- I am running with identical parameters on both my laptop and the HPC.
- I'm not trying to run it as a distributed memory program.
- I don't have any insights into how the HPC does file I/O. But I can double-check both this question and the issue of running up against limits with the HPCC tech support from my university.
- I have described this problem to the HPC tech support, but they had no idea why this error message popped up.

**Comment by rheiland** [comment]
What release of PhysiCell are you using - the latest, 1.11.0 ?  (see VERSION.txt in your root directory)

**Comment by tony1747** [comment]
The version I'm using is 1.9.1. 

I didn't try the original cancer immune 3D on HPCC. What you have is the same as the ones I have in the beginning. The error message shows up after about 30 images (or even later).

**Comment by tony1747** [comment]
I will try to use the latest version on HPCC and see if that makes any difference. 

As for the model, our project requires to build a 3D instead of a 2D model, so I chose to start with 3D model. 

For the cells' positions, I used the given data to assign positions for every cancer cell initially. In this case, red cells denote the cancer cells while grey cells denote healthy cells (tissue). After a certain period of time, I introduced immune cells (colored in purple) in the middle as a spheroid. 

By the way, I believe the parameters that I had in this setting are incorrect. In general, every cell should be adjacent to each other in the beginning instead of splitting up into several neighborhoods. I should have used a smaller number for cell to cell adhesion strength and bigger number for cell to cell repulsion strength. 

I will talk to the HPCC tech support again with your updates, and I will follow up if the problem is still unsolved.
